perma_web/perma/tasks.py

```diff
@@ -592,45 +593,115 @@ def update_stats():
         current_week.save()
 
 
-@shared_task(
-    bind=True,
-    default_retry_delay=10*60) # 10 minute delay between retries
+@shared_task(bind=True)
+def delete_from_internet_archive(self, link_guid):
+    identifier = settings.INTERNET_ARCHIVE_IDENTIFIER_PREFIX + link_guid
```

Comment:
Does this need `if not settings.UPLOAD_TO_INTERNET_ARCHIVE: return` at the top like the other function?